code/game/objects/items/devices/pseudocider.dm

```diff
@@ -0,0 +1,245 @@
+/obj/item/pseudocider
+	name = "syndicate pseudocider"
+	desc = "A syndicate device that triggers upon taking damage, making you invisible and leaving behind a fake body."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "pocketwatch-closed"
+	w_class = WEIGHT_CLASS_SMALL
+	actions_types = list(/datum/action/item_action/toggle_pseudocider)
+	var/active = FALSE
+	var/mob/living/carbon/fake_corpse
+	COOLDOWN_DECLARE(fake_death_timer)
+	var/fake_death_cooldown = 20 SECONDS
+
+/obj/item/pseudocider/Initialize(mapload)
+	. = ..()
+	// Meant to work from the pocket as well
+	RegisterSignals(src, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_PICKUP), PROC_REF(assign_user))
+	RegisterSignal(src, COMSIG_ITEM_DROPPED, PROC_REF(drop_user))
+	RegisterSignal(src, COMSIG_ITEM_ATTACK_SELF, PROC_REF(self_attack))
+
+/obj/item/pseudocider/proc/assign_user(datum/source, mob/living/carbon/user)
+	SIGNAL_HANDLER
+	if(!istype(user))
+		return
+	RegisterSignal(user, COMSIG_MOB_APPLY_DAMAGE, PROC_REF(fake_death))
+
+/obj/item/pseudocider/proc/drop_user(datum/source, mob/living/carbon/user)
+	SIGNAL_HANDLER
+	UnregisterSignal(user, COMSIG_MOB_APPLY_DAMAGE)
+
+/obj/item/pseudocider/proc/self_attack(datum/source, mob/user)
+	SIGNAL_HANDLER
+	if(!iscarbon(user))
+		return
+	if(HAS_TRAIT(user, TRAIT_NOINTERACT))
+		return
+	if(!COOLDOWN_FINISHED(src, fake_death_timer) && !active)
+		to_chat(user, span_notice("\The [src] refuses to open! Wait [COOLDOWN_TIMELEFT(src, fake_death_timer)/10] more seconds!"))
+		return
+	active = !active
+	if(active)
+		icon_state = "pocketwatch-open"
+	else
+		icon_state = "pocketwatch-closed"
+	update_appearance(UPDATE_ICON)
+
+/obj/item/pseudocider/proc/fake_death(mob/living/carbon/copied_mob, damage, damagetype, def_zone)
+	SIGNAL_HANDLER
+	if(HAS_TRAIT(copied_mob, TRAIT_NOINTERACT))
+		return
+	if(!active)
+		return
+	if(!COOLDOWN_FINISHED(src, fake_death_timer))
+		return
+	if(damage <= 0)
+		return
+	if(!istype(copied_mob))
+		return
+	if(copied_mob.stat > CONSCIOUS)
+		return
+	var/turf/copy_location = get_turf(copied_mob)
+	if(!istype(copy_location))
+		return
+
+	COOLDOWN_START(src, fake_death_timer, fake_death_cooldown) // sanity
+
+	fake_corpse = new copied_mob.type
+	if(copied_mob.dna?.species)
+		INVOKE_ASYNC(fake_corpse, TYPE_PROC_REF(/mob,set_species), new copied_mob.dna.species.type)
+		fake_corpse.dna.features = LAZYCOPY(copied_mob.dna.features)
+
+	// Set up the mob to be identical
+	if(ishuman(fake_corpse) && ishuman(copied_mob))
```

Comment:
Do we not have any helpers to do this stuff?
If not, can you make these all helpers?